### ISSUE TITLE
Differentiable softmax

### ISSUE THREAD

**Issue by gitE0Z9** [gitE0Z9/pytorch-implementations#63]

**Comment by gitE0Z9** [comment]
https://aclanthology.org/P16-1186.pdf
